eng/Compilers.props

```diff
@@ -8,12 +8,13 @@
 
   <PropertyGroup Condition="!($(MSBuildProjectName.EndsWith('Tests', StringComparison.OrdinalIgnoreCase)))">
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <NoWarn></NoWarn> <!-- Make sure the list is empty - by default it is not -->
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <NoWarn>$(NoWarn.Replace("1591", ""))</NoWarn> <!-- Make sure CS1591 is not in the list (by default it is) -->
```

Comment:
can you write what 1591 is?

Reply:
Already fixed with joperezr's commit. It's all about this: "Missing documentation for publicly visible type"